flow_deps/README.MD

```diff
@@ -0,0 +1,13 @@
+### What is this folder?
+This folder houses the files we pull in from the Estuary/flow project. 
```

Comment:
Since the name of the repository is _estuary/flow_, you may want to replace the capital _E_ with its lowercase counterpart.